homeassistant/components/dynalite/cover.py

```diff
@@ -32,9 +31,8 @@ class DynaliteCover(DynaliteBase, CoverEntity):
     def device_class(self) -> str:
         """Return the class of the device."""
         dev_cls = self._device.device_class
-        if dev_cls in DEVICE_CLASSES:
-            return dev_cls
-        return DEFAULT_COVER_CLASS
+        assert dev_cls in DEVICE_CLASSES
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5462445)